cooperator/models/res_partner.py

```diff
@@ -283,3 +285,21 @@ def create_cooperative_membership(self, company_id):
                 }
             )
         return result
+
+    def get_share_quantities(self):
```

Comment:
i think that this should better be defined on `cooperative.membership`, as this is different per company.